src/main/java/uk/gov/ch/pocappointmentapi/AppointmentController.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.ch.pocappointmentapi;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AppointmentController {
```

Comment:
We could use JAX-RS annotations here for added portability - you may need additional dependencies and configuration for the Spring Boot plugin in pom.xml to support this, though.

Reply:
We could, but this is just a POC so trying to keep it as simple as possible.